server/base/src/main/java/org/apache/accumulo/server/fs/VolumeManagerImpl.java

```diff
@@ -385,10 +385,10 @@ public boolean isReady() throws IOException {
   }
 
   @Override
-  public Path matchingFileSystem(Path source, String[] options) {
+  public Path matchingFileSystem(Path source, Set<String> options) {
     try {
       if (ViewFSUtils.isViewFS(source, hadoopConf)) {
-        return ViewFSUtils.matchingFileSystem(source, options, hadoopConf);
+        return ViewFSUtils.matchingFileSystem(source, options.toArray(new String[0]), hadoopConf);
```

Comment:
Could make ```ViewFSUtils``` take a Set as well.  Looks like this is the only class that uses it.

Reply:
I'm going to deal with this class separately... I think we can rip out the viewfs support tooling... since we explicitly ban it.